index.html

```diff
@@ -4713,6 +4713,25 @@ <h2>Verification Method Revocation</h2>
 could discover a revoked key and use it to make cryptographically
 verifiable statements with a simulated date in the past.
 <br/><br/>
+For example, for a DID method that supports the `versionId` query parameter as
+well as the `updated` and `nextUpdate` DID document metadata properties, a
+verifier can validate a signature or proof of a revoked key if and only if 
+all of the following are true:
+  <ol>
+    <li>
+The proof or signature contains the `versionId` to refer to the DID document
+that was used at the point the signature or proof was created.
```

Comment:
```suggestion
The proof or signature contains the `versionId` to refer to the DID document
that was used at the point the signature or proof was created.
```

This is not necessarily true -- you can store that information out of band, it doesn't have to be in a signature. For example, if you receive a signature you can store the version ID of the DID Document you got when you checked the signature in the first place. When the key is revoked in the future, you know (without using the proof/signature information) what version ID to retrieve. The same is true for the `hl` property -- stored out of band... and so you wouldn't need any of these properties to validate a signature for a revoked key.

Reply:
See https://github.com/w3c/did-core/pull/668#issuecomment-783232155